generative_ai/extraction_test.py

```diff
@@ -12,13 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
+
 import backoff
 from google.api_core.exceptions import ResourceExhausted
 
 import extraction
 
 
+_PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
+_LOCATION = "us-central1"
```

Comment:
nit: `us-central` region is used often in our tests, is it possible to use a different region? Fine to leave it for now.